src/components/JaegerIntegration/JaegerResults/JaegerTraceTitle.tsx

```diff
@@ -18,12 +18,8 @@ export class JaegerTraceTitle extends React.Component<Props> {
     return (
       <CardHeader style={{ backgroundColor: PfColors.Black200, height: '50px' }}>
         <Text component={TextVariants.h3} style={{ margin: 0, position: 'relative' }}>
-          {formattedTrace.name}
-          <Tooltip content={<>{traceID}</>}>
-            <span style={{ color: PfColors.Black600, paddingLeft: '10px', fontSize: '14px' }}>
-              {traceID.slice(0, 7)}
-            </span>
-          </Tooltip>
+          Trace details: {formattedTrace.name}
+          <span style={{ color: PfColors.Black600, paddingLeft: '10px', fontSize: '14px' }}>{traceID}</span>
           {this.props.onClickLink !== '' && (
```

Comment:
Great one, not need slide now
![Screenshot from 2020-09-28 13-59-28](https://user-images.githubusercontent.com/3019213/94429617-e001cf00-0192-11eb-8572-942a9490288c.png)

Reply:
What do you think about to have a link in the ID to the Trace in Jaeger too @jotak ? or use the id and remove the "View Trace in Tracing"

Reply:
I would prefer to keep external links very explicit.
But I didn't notice it was on the left, perhaps it should be moved next to "View on Graph".